emulator/index.d.ts

```diff
@@ -0,0 +1,95 @@
+declare module "android-emulator-webrtc/emulator" {
```

Comment:
This file is _already_ this module by its location.

Reply:
https://github.com/google/android-emulator-webrtc/pull/46#discussion_r1814808585